jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -2578,7 +2589,7 @@ public final boolean isRegisteredAsParallelCapable() {
 }
 /*[ENDIF] JAVA_SPEC_VERSION >= 9 */
 
-/*[IF JAVA_SPEC_VERSION >= 19]*/
+/*[IF (JAVA_SPEC_VERSION >= 19) & (JAVA_SPEC_VERSION < 24)]*/
```

Comment:
Please use the "between" idiom (like `Thread.java`):
```java
/*[IF (19 <= JAVA_SPEC_VERSION) & (JAVA_SPEC_VERSION < 24)]*/
```

Reply:
Updated